packages/dds/task-manager/src/test/taskManager.fuzz.spec.ts

```diff
@@ -210,24 +218,30 @@ function makeReducer(loggingInfo?: LoggingInfo): Reducer<Operation, FuzzTestStat
 	return withLogging(reducer);
 }
 
-function assertEqualTaskManagers(a: ITaskManager, b: ITaskManager) {
+/* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any */
+function assertEqualTaskManagers(a: ITaskManager, b: ITaskManager): void {
 	const queue1 = (a as any).taskQueues;
 	const queue2 = (b as any).taskQueues;
 	assert.strictEqual(queue1.size, queue2.size, "The number of tasks queues are not the same");
 	for (const [key, val] of queue1) {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-call
 		const testVal = queue2.get(key);
 		if (testVal === undefined) {
 			assert(val === undefined, "Task queues are not both undefined");
 			continue;
 		}
 		assert.strictEqual(testVal.length, val.length, "Task queues are not the same size");
 		if (testVal.length > 0) {
-			testVal.forEach((task: string, index: number) => {
-				assert.strictEqual(task, val[index], `Task queues are not identical`);
-			});
+			const testValArr = testVal as string[];
+			const valArr = val as string[];
```

Comment:
Could we avoid these by typing `queue1` and `queue2` above as `: Map<string, string[]>` like the property we're hacking our way into? Would also avoid the eslint disable on 227, I think.

Reply:
Done.